requirements.txt

```diff
@@ -6,7 +6,7 @@ faker~=20.1.0
 colorama~=0.4.6
 pytest
 time-machine==2.13.0
-tiktoken~=0.5.2
+tiktoken
```

Comment:
About Antony's issue with the packages: we could simulate a fresh-new installation from time to time. I think the order of packages in the requirements file is important, and we can have it ordered in a way that conflicts don't show upd.